.github/workflows/build.yml

```diff
@@ -58,15 +59,15 @@ jobs:
           echo "COMMIT_SHA=$COMMIT_SHA" >> $GITHUB_ENV
           echo "COMMIT_MESSAGE=$(git log -n 1 --pretty=format:%s $COMMIT_SHA)" >> $GITHUB_ENV
     outputs:
-      commit_label: "${{ env.COMMIT_SHA }}: ${{ env.COMMIT_MESSAGE }}"
+      commit_label: '${{ env.COMMIT_SHA }}: ${{ env.COMMIT_MESSAGE }}'
```

Comment:
No action required: I guess those are some automatic Prettier (or similar) changes?

Reply:
Yup, I'll leave it (I'm sorry for the messed up git history).